tuf/api/metadata.py

```diff
@@ -104,9 +94,12 @@ class Metadata(Generic[T]):
     "[Root]" is not validated at runtime (as pure annotations are not available
     then).
 
-    Attributes:
-        signed: A subclass of Signed, which has the actual metadata payload,
-            i.e. one of Targets, Snapshot, Timestamp or Root.
+    *All parameters named below are not just constructor arguments but also
+    instance attributes.*
+
+    Args:
+        signed: The actual metadata payload, i.e. one of Targets, Snapshot,
+            Timestamp or Root.
         signatures: An ordered dictionary of keyids to Signature objects, each
```

Comment:
nit: I couldn't figure out why `OrderedDictionary` is not displayed as a type here but it is for `roles` in `class Delegations`

Reply:
huh... I thought autodoc wasn't able to handle the type-as-string `"OrderedDict"` but you are right that the same thing works in Delegations. How weird

Reply:
huh, the types do appear on my local build -- I think this might be a missing feature in the version of sphinx/autodoc that RTD uses: at least Generics support had been improved in recent releases.